README.md

```diff
@@ -45,7 +45,7 @@ The prerequisites of building hpc-stack are:
 - [Lmod](https://lmod.readthedocs.io/en/latest/) - An Environment Module System
 - CMake and make
 - wget and curl
-- git
+- git, git-lsf
```

Comment:
Should be git-lfs. 
Should correct in a future pr.

Reply:
Corrected